.github/workflows/app-publish.yaml

```diff
@@ -0,0 +1,73 @@
+name: App Publish
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    environment: release
+    if: ${{ github.repository == 'jellyfin/jellyfin-android' }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+
+      - name: Set JELLYFIN_VERSION
+        run: echo "JELLYFIN_VERSION=$(echo ${GITHUB_REF#refs/tags/v} | tr / -)" >> $GITHUB_ENV
+
+      - name: Build Signed Release
+        env:
+          KEYSTORE: ${{ secrets.KEYSTORE }}
+          KEYSTORE_PASSWORD: ${{ secrets.KEYSTORE_PASSWORD }}
+          KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
+          KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
+        run: ./gradlew --no-daemon --info assembleRelease versionTxt
+
+      - name: Split APK release types
+        run: |
+          mkdir -p build/jellyfin-publish/release-libre build/jellyfin-publish/release-proprietary;
+          mv app/build/outputs/apk/*/*/jellyfin-android-*-libre-release.apk build/jellyfin-publish/release-libre/;
+          mv app/build/outputs/apk/*/*/jellyfin-android-*-proprietary-release.apk build/jellyfin-publish/release-proprietary/;
+          mv app/build/version.txt build/jellyfin-publish/;
+
+      - name: Upload Release Artifacts
+        uses: alexellis/upload-assets@0.3.0
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          asset_paths: >-
+            '["build/jellyfin-publish/release-libre/*",
+            "build/jellyfin-publish/release-proprietary/*",
+            "build/jellyfin-publish/version.txt"]'
+
+      - name: Upload to repo.jellyfin.org
+        uses: burnett01/rsync-deployments@4.1
+        with:
+          switches: -rltgoDzvO --delete --exclude='*' --include='**/*.apk' --include='*.txt'
+          path: build/jellyfin-publish/
+          remote_path: /srv/repository/releases/client/android/versions/v${{ env.JELLYFIN_VERSION }}
+          remote_host: ${{ secrets.DEPLOY_HOST }}
+          remote_user: ${{ secrets.DEPLOY_USER }}
+          remote_key: ${{ secrets.DEPLOY_KEY }}
```

Comment:
- [ ] These need to be added (preferably on organization level so we can re-use it in the ATV app later).

_(mark when added)_